static/include/components/board-attr-config.md

```diff
@@ -0,0 +1,60 @@
+Copy and paste the following attribute template into your board's **Attributes** box.
+Then remove and fill in the attributes as applicable to your board, according to the table below.
+
+{{< tabs >}}
+{{% tab name="Attributes template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "analogs": [
```

Comment:
These are all going away and we are not having a standard config for the board component. The common config is too tied to the raspberry pi, and we've already broken it for custom-linux.